.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -49,6 +49,7 @@ stages:
             if [[ "${IMG_SKU}" == "20_04-lts-cvm" ]]; then SKU_NAME="${SKU_NAME}CVM"; fi && \
             if [[ "${IMG_SKU}" == *"minimal"* ]]; then SKU_NAME="${SKU_NAME}minimal"; fi && \
             if [[ "${ENABLE_TRUSTED_LAUNCH}" == "True" ]]; then SKU_NAME="${SKU_NAME}TL"; fi && \
+            if [[ ${OS_VERSION} == "V2" && ${OS_SKU} == "CBLMariner" && ${ENABLE_CGROUPV2,,} == "true" ]]; then SKU_NAME="${SKU_NAME}cgroupv2"; fi && \
```

Comment:
The open question here is whether we need to introduce a new SKU_NAME for cgroupv2 azure linux aks images or not?

Reply:
per offline discussion we decided to enable cgroupsv2 for all existing marinerv2 SKUs, which will be built to the new AzureLinux gallery

Reply:
okay we making new SKUs :D